Sources/OAuth2Client/NXOAuth2Connection.m

```diff
@@ -187,6 +187,8 @@ - (NSURLConnection *)createConnection;
     
     if (oauthAuthorizationHeader) {
         [startRequest setValue:oauthAuthorizationHeader forHTTPHeaderField:@"Authorization"];
+        // some services require the access token in the "t_auth_token" header field
+        [startRequest setValue:client.accessToken.accessToken forHTTPHeaderField:@"t_auth_token"];
```

Comment:
Might be that some services require this, but this is not part of the IETF standard in any RFC that I'm aware of, or am I missing something?